src/libraries/System.Reflection.Emit/tests/PersistableAssemblyBuilder/AssemblySaveTypeBuilderAPIsTests.cs

```diff
@@ -5,6 +5,8 @@
 using System.IO;
 using System.Runtime.InteropServices;
 using System.Text;
+using Microsoft.VisualBasic.FileIO;
```

Comment:
nit: Looks like the VisualBasic was added accidentally.

Reply:
Good catch